exchange/gdpr_test.go

```diff
@@ -47,28 +49,56 @@ func TestExtractGDPR(t *testing.T) {
 			wantGDPR:    gdpr.SignalAmbiguous,
 			wantError:   true,
 		},
+		{
+			description: "Regs Ext GDPR = null, GPPSID has tcf2",
+			giveRegs:    &openrtb2.Regs{Ext: json.RawMessage(`{"gdpr": null}`), GPPSID: []int8{2}},
+			wantGDPR:    gdpr.SignalYes,
+		},
+		{
+			description: "Regs Ext GDPR = 1, GPPSID has uspv1",
+			giveRegs:    &openrtb2.Regs{Ext: json.RawMessage(`{"gdpr": 1}`), GPPSID: []int8{6}},
+			wantGDPR:    gdpr.SignalNo,
+		},
+		{
+			description: "Regs Ext GDPR = 0, GPPSID has tcf2",
+			giveRegs:    &openrtb2.Regs{Ext: json.RawMessage(`{"gdpr": 0}`), GPPSID: []int8{2}},
+			wantGDPR:    gdpr.SignalYes,
+		},
+		{
+			description: "Regs Ext is nil, GPPSID has tcf2",
+			giveRegs:    &openrtb2.Regs{GPPSID: []int8{2}},
+			wantGDPR:    gdpr.SignalYes,
+		},
+		{
+			description: "Regs Ext is nil, GPPSID has uspv1",
+			giveRegs:    &openrtb2.Regs{GPPSID: []int8{6}},
+			wantGDPR:    gdpr.SignalNo,
+		},
 	}
 
 	for _, tt := range tests {
-		bidReq := openrtb2.BidRequest{
-			Regs: tt.giveRegs,
-		}
+		t.Run(tt.description, func(t *testing.T) {
+			bidReq := openrtb2.BidRequest{
+				Regs: tt.giveRegs,
+			}
 
-		result, err := extractGDPR(&bidReq)
-		assert.Equal(t, tt.wantGDPR, result, tt.description)
+			result, err := extractGDPR(&bidReq)
+			assert.Equal(t, tt.wantGDPR, result, tt.description)
```

Comment:
There is no longer a need to include the descriptions as arguments to the assert methods now that the test is contained in a t.Run call.